.ci/ciao_down_setup_env_ubuntu.sh

```diff
@@ -0,0 +1,109 @@
+source "$cidir/../test-versions.txt"
+
+echo "enable Nested Virtualization"
+sudo modprobe -r kvm_intel
+sudo modprobe kvm_intel nested=1
+
+echo "Add clear containers sources to apt list"
+sudo sh -c "echo 'deb http://download.opensuse.org/repositories/home:/clearlinux:/preview:/clear-containers-2.1/xUbuntu_16.10/ /' >> /etc/apt/sources.list.d/cc-oci-runtime.list"
+curl -fsSL http://download.opensuse.org/repositories/home:clearlinux:preview:clear-containers-2.1/xUbuntu_16.10/Release.key | sudo -E apt-key add -
+sudo -E apt update
+
+echo "Install Build-Essential"
+sudo -E apt install -y build-essential python pkg-config zlib1g-dev libglib2.0-dev
+
+echo "Install clear containers dependencies"
+sudo -E apt install -y libtool automake autotools-dev autoconf bc
+
+echo "Install chronic"
+sudo -E apt install -y moreutils
+
+echo "Install rpm2cpio"
+chronic sudo -E apt install -y rpm2cpio
+
+echo "Update apt repositories"
+chronic sudo -E apt update
+
+echo "Install qemu-lite binary"
+chronic sudo -E apt install -y --force-yes qemu-lite
+
+echo "Install qemu Q35 binary"
+chronic sudo -E apt install -y libcap-ng-dev libpixman-1-dev libcap-dev libattr1-dev
+git clone --branch qemu-lite-v2.9.0 https://github.com/clearcontainers/qemu.git --depth 1
+qemu_dir="qemu"
+pushd ${qemu_dir}
+git checkout qemu-lite-v2.9.0
+./configure --disable-tools --disable-libssh2 --disable-tcmalloc --disable-glusterfs        \
+	--disable-seccomp --disable-{bzip2,snappy,lzo} --disable-usb-redir --disable-libusb \
+	--disable-libnfs --disable-tcg-interpreter --disable-debug-tcg --disable-libiscsi   \
+	--disable-rbd --disable-spice --disable-attr --disable-cap-ng --disable-linux-aio   \
+	--disable-brlapi --disable-vnc-{jpeg,png,sasl} --disable-rdma --disable-bluez       \
+	--disable-fdt --disable-curl --disable-curses --disable-sdl --disable-gtk           \
+	--disable-tpm --disable-vte --disable-vnc --disable-xen --disable-opengl            \
+	--disable-slirp --enable-trace-backend=nop --enable-virtfs --enable-attr            \
+	--enable-cap-ng --target-list=x86_64-softmmu
+make -j$(nproc)
+sudo -E PATH=$PATH sh -c "make install"
+sudo -E mv $(which qemu-system-x86_64) /usr/bin/qemu-q35-system-x86_64
+popd
+rm -rf ${qemu_dir}
+
+echo "Install CRI-O dependencies"
+sudo -E apt install -y libglib2.0-dev   \
+	libseccomp-dev libapparmor-dev  \
+	libgpgme11-dev libdevmapper-dev \
+	btrfs-tools util-linux
+
+clear_release=$(curl -sL https://download.clearlinux.org/latest)
+cc_img_path="/usr/share/clear-containers"
+
+"${cidir}/install_clear_image.sh" ${clear_release} "${cc_img_path}"
+
+bug_url="https://github.com/clearcontainers/runtime/issues/91"
+echo -e "\nWARNING:"
+echo "WARNING: Using backlevel kernel version ${kernel_version} due to bug ${bug_url}"
+echo -e "WARNING:\n"
+
+"${cidir}/install_clear_kernel.sh" ${kernel_clear_release} ${kernel_version} "${cc_img_path}"
+
+echo "Install bison binary"
+chronic sudo -E apt install -y bison
+
+echo "Install ostree"
```

Comment:
We can install libostree more easily inside ciao-down because it is xenial. There is a ppa here https://launchpad.net/~alexlarsson/+archive/ubuntu/flatpak that contains libostree.